test/tracecountheader_test.rb

```diff
@@ -1,14 +1,21 @@
 require 'helper'
 require 'ddtrace'
 require 'ddtrace/tracer'
+require 'stringio'
 require 'thread'
 require 'webrick'
 
 class TraceCountHeaderTest < Minitest::Test
   TEST_PORT = 6218
 
   def setup
-    @server = WEBrick::HTTPServer.new Port: TEST_PORT
+    @log_buf = StringIO.new
+    log = WEBrick::Log.new @log_buf
+    access_log = [
+      [@log_buf, WEBrick::AccessLog::COMBINED_LOG_FORMAT]
+    ]
+
+    @server = WEBrick::HTTPServer.new Port: TEST_PORT, Logger: log, AccessLog: access_log
```

Comment:
Exactly, else tests where polluted by random Webrick output, not really relevant. Can be useful when debugging the test, else, useless.